Jamulus.pro

```diff
@@ -93,10 +93,29 @@ win32 {
 
     # replace ASIO with jack if requested
     contains(CONFIG, "jackonwindows") {
-        message(Using Jack instead of ASIO.)
-
-        !exists("C:/Program Files (x86)/Jack/includes/jack/jack.h") {
-            message(Warning: jack.h was not found at the usual place, maybe jack is not installed)
+        contains(QT_ARCH, "i386") {
+            !exist("C:/Program Files (x86)") {
+                message ("Cross compilation build")
+                programfilesdir = "C:/Program Files (x86)"
+                !exists("$${"programfilesdir"}/JACK2/include/jack/jack.h") {
+                    message("Warning: jack.h was not found at the usual place, maybe JACK2 32 bits is not installed to cross compile.")
+                }
+            }
+            !exist("C:/Program Files") {
+                message("Native i386 build")
+                programfilesdir = "C:/Program Files"
+                !exists("$${"programfilesdir"}/JACK2/include/jack/jack.h") {
```

Comment:
Should this rather be an `else` branch to the previous check? Otherwise, both checks would run, wouldn't they? I guess that would not be intended.

Reply:
If QT_ARCH is set to i386, it is a x86 native or a x64 cross compile for 32 bit. It should work like that. lets see wha append when @henkdegroot builds.

Reply:
Adding an else at last.